dataPipelines/gc_thumbnails/utils.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+import fitz
+import typing as t
+
+class ThumbnailsCreator:
+    def __init__(
+        self,
+        input_directory: t.Union[str, Path],
+        output_directory: t.Union[str, Path]
+    ):
+        self.input_directory = Path(input_directory).absolute()
+        self.output_directory = Path(output_directory).absolute()
+
+    def generate_thumbnails(self):
+        print('generating')
+        p = self.input_directory
+        if not self.output_directory.exists():
```

Comment:
no need to check `Path.mkdir(exist_ok=True)` will do that already

Reply:
ahhhh good catch